gnomad/resources/grch37/reference_data.py

```diff
@@ -10,6 +10,54 @@
     import_sites_vcf,
 )
 
+
+def _import_gtex_rsem(gtex_path: str, meta_path: str) -> hl.MatrixTable:
+    """
+    Import GTEx RSEM data from expression data and sample attributes file.
+
+    .. note::
+        Files are downloaded from https://www.gtexportal.org/home/downloads/adult-gtex.
+        We get the transcript TPM under Bulk tissue expression and sample attributes
+        under Metadata. The transcript TPM file is expected to have transcript
+        expression data, with transcript IDs as the first column and gene IDs as the
+        second column.
+    :param gtex_path: Path to the GTEx RSEM file.
+    :param meta_path: Path to the GTEx sample attributes file.
+    :return: Matrix Table with GTEx RSEM data with tissue information.
+    """
+    meta_ht = hl.import_table(meta_path, force_bgz=True, impute=True)
+    meta_ht = meta_ht.key_by("SAMPID")
+
+    mt = hl.import_matrix_table(
+        gtex_path,
+        row_key="transcript_id",
+        row_fields={"transcript_id": hl.tstr, "gene_id": hl.tstr},
+        entry_type=hl.tfloat64,
+        force_bgz=True,
+        min_partitions=1000,
+    )
+
+    mt = mt.rename({"x": "transcript_tpm"})
+
+    # GTEx data has gene IDs and transcript IDs with version numbers, we need
+    # to remove the version numbers so that it can later be joined with the
+    # variant Table
+    mt = mt.annotate_cols(
+        tissue=meta_ht[mt.col_id]
+        .SMTSD.replace(" ", "")
+        .replace("-", "_")
+        .replace("\\(", "_")
+        .replace("\\)", "")
+    )
+    mt = mt.key_rows_by()
```

Comment:
Would it work to just not use `row_key="transcript_id"` when importing?

Reply:
When it's key, I can't remove the version number.

Reply:
Yeah, but I think without `row_key="transcript_id"` in the import it won't already be keyed

Reply:
You're right, I had row_key in my test code.